builder/src/main/java/cz/xtf/builder/builders/PodBuilder.java

```diff
@@ -50,10 +51,22 @@ public ContainerBuilder container(String name) {
         return getContainerBuilder(name);
     }
 
+    public ContainerBuilder initContainer() {
+        return getInitContainerBuilder(getName() + "-init");
```

Comment:
In case more initContainer is added using this method then those would collide with names, rith? What about to add some short random hash here or counter?

Reply:
furthermore with these changes I wouldnt be able to call `container()/initcontainer()` twice to get the same container

Reply:
>  There is the same problem with unparametrized container() method which should not be here ideally (only the parametrized version)

perhaps you're right. I just see it as "my deployment will only have one container and IDC about name" method